packages/teams-js/test/public/meeting.spec.ts

```diff
@@ -1283,4 +1283,67 @@ describe('meeting', () => {
       expect(response).toBe(meetingReaction);
     });
   });
+  describe('setVisibilityInfo', () => {
+    let shareUrl = 'https://test.com';
+    let shareInformation: meeting.appShareButton.ShareInformation = {
+      isVisible: false,
+      shareUrl: shareUrl,
+    };
+    it('meeting.appShareButton.setVisibilityInfo should not allow calls before initialization', () => {
+      expect(() => meeting.appShareButton.setVisibilityInfo(shareInformation)).toThrowError(
+        'The library has not yet been initialized',
+      );
+    });
+    const allowedContexts = [FrameContexts.sidePanel];
+    Object.values(FrameContexts).forEach((context) => {
+      if (allowedContexts.some((allowedContext) => allowedContext === context)) {
+        it(`should successfully set shareInformation. context: ${context}`, async () => {
+          await framelessPlatformMock.initializeWithContext(context);
+          meeting.appShareButton.setVisibilityInfo(shareInformation);
+          const toggleAppShareButtonMessage = framelessPlatformMock.findMessageByFunc(
+            'meeting.appShareButton.setVisibilityInfo',
+          );
+          expect(toggleAppShareButtonMessage).not.toBeNull();
+          expect(toggleAppShareButtonMessage.args.length).toBe(1);
+          expect(toggleAppShareButtonMessage.args[0]).toStrictEqual(shareInformation);
+        });
+
+        it(`should successfully set false isVisible and shareUrl to be null. context: ${context}`, async () => {
+          await framelessPlatformMock.initializeWithContext(context);
+          shareInformation.shareUrl = null;
```

Comment:
I won't type this every place you do it in this file, but since shareUrl is optional you can just omit it instead of setting it to null